can/io/blf.py

```diff
@@ -0,0 +1,263 @@
+"""
+Implements support for BLF (Binary Logging Format) which is a proprietary
+CAN log format from Vector Informatik GmbH.
+
+No offical specification is available. The implementation is based on the works
+by Toby Lorenz (https://bitbucket.org/tobylorenz/vector_blf).
+The file starts with a header. The rest is one or more "log containers"
+which consists of a header and some zlib compressed data, usually up to 128 kB
+of uncompressed data each. This data contains the actual CAN messages and other
+objects types.
+"""
```

Comment:
Typo in *official*. Suggest we add a comment about the license of the Vector BLF library too. Something like:

> No official specification of the binary logging format is available. This implementation is based on Toby Lorenz' C++ library "Vector BLF" which is licenced under GPLv3. https://bitbucket.org/tobylorenz/vector_blf

Reply:
Hi, 

If this implementation is based on Toby Lorenz' C++ library "Vector BLF" which is licensed under GPLv3, do you think that there is a chance that the license of this file (if this file is licensed under GPL v3.0 too) could affect licensing of other files too and make the whole component GPL v3.0?

Reply:
It's just used as a reference for the format, none of the code is used. So I would say no if I understand the license correctly.

Reply:
Ok